plugins/catalog/src/alpha/DefaultEntityContentLayout.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import { EntityContentLayoutProps } from '@backstage/plugin-catalog-react/alpha';
+import { EntitySwitch } from '../components/EntitySwitch';
+import {
+  EntityOrphanWarning,
+  isOrphan,
+} from '../components/EntityOrphanWarning';
+import {
+  EntityRelationWarning,
+  hasRelationWarnings,
+} from '../components/EntityRelationWarning';
+import {
+  EntityProcessingErrorsPanel,
+  hasCatalogProcessingErrors,
+} from '../components/EntityProcessingErrorsPanel';
+import { HorizontalScrollGrid } from '@backstage/core-components';
+
+const useStyles = makeStyles(theme => ({
+  peekCard: {
+    flex: '0 0 auto',
+  },
+  [theme.breakpoints.up('sm')]: {
+    infoArea: {
+      order: 1,
+      position: 'sticky',
+      top: -16,
```

Comment:
I thinking a css grid would simplify the layout tbh, MUI grid isn't super good at this kind of stuff :p